### PR TITLE
correction to example 1 in Set-AzEventHub.md

### PR DESCRIPTION
**Pull request by atul-ram**
Corrected input object variable in example 1.

## Description

This correction is required as the present example ends with an error.
[error]Parameter set cannot be resolved using the specified named parameters. 

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by msJinLei** [comment]
@atul-ram Are you a MS FTE?

**Comment by atul-ram** [comment]
> @atul-ram Are you a MS FTE?

No

File: src/EventHub/EventHub/help/Set-AzEventHub.md [hunk 1]
```diff
@@ -45,10 +45,10 @@ PS C:\> $createdEventHub.CaptureDescription.Destination.Name = "EventHubArchive.
 PS C:\> $createdEventHub.CaptureDescription.Destination.BlobContainer = "container"
 PS C:\> $createdEventHub.CaptureDescription.Destination.ArchiveNameFormat = "{Namespace}/{EventHub}/{PartitionId}/{Year}/{Month}/{Day}/{Hour}/{Minute}/{Second}"
 PS C:\> $createdEventHub.CaptureDescription.Destination.StorageAccountResourceId = "/subscriptions/{SubscriptionId}/resourceGroups/MyResourceGroupName/providers/Microsoft.ClassicStorage/storageAccounts/arjunteststorage"
-PS C:\> Set-AzEventHub -ResourceGroupName MyResourceGroupName -Namespace MyNamespaceName -Name MyEventHubName -InputObject MyCreatedEventHub -messageRetentionInDays 4 -partitionCount 2
```
Comment (msJinLei) [inline src/EventHub/EventHub/help/Set-AzEventHub.md:48]: Why do you remove the "-messageRetentionInDays 4 -partitionCount 2"

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/EventHub/EventHub/help/Set-AzEventHub.md (hunk 1 above)
Comment (atul-ram) [reply src/EventHub/EventHub/help/Set-AzEventHub.md:48]: in order to change the 'Capture description' properties for an existing  **MyEventHubName** .
the proposal is to use minimum options.  if you add these two values.  
you keep getting error : - 
[error]Parameter set cannot be resolved using the specified named parameters.

**Review by msJinLei — APPROVED** [review]

**Comment by erich-wang** [comment]
Force merge it because of CI issue.

**Pull request merged**